pkg/interfaces/contracts/pool-stable/IStablePool.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import { IBasePool } from "../vault/IBasePool.sol";
+
+struct StablePoolDynamicData {
+    uint256[] liveBalances;
+    uint256[] tokenRates;
+    uint256 staticSwapFeePercentage;
+    uint256 totalSupply;
+    uint256 bptRate;
+    uint256 amp;
```

Comment:
Should this be amplificationParameter to match the code? Maybe just "amp" is ok, but seems short.